Source/UI/NewUI/PaymentForm/View/ExpiryDateView.swift

```diff
@@ -0,0 +1,130 @@
+import UIKit
+import Checkout
+
+protocol ExpiryDateViewDelegate: AnyObject {
+  func update(expiryDate: ExpiryDate)
+}
+
+public final class ExpiryDateView: UIView {
+  weak var delegate: ExpiryDateViewDelegate?
+  
+  /// 5 is the expected text count, for example "11/35".
+  private let dateFormatTextCount = 5
+  
+  private(set) var style: CellTextFieldStyle?
+  
+  private lazy var cardValidator: CardValidator = {
+    CardValidator(environment: .production)
```

Comment:
this needs be injected in from further up. The environment must match the actual environment